src/server/middleware/sendHtml.js

```diff
@@ -57,7 +57,27 @@ export function safeSend(res, ...payload) {
   }
 }
 
-export function renderStaticErrorPage(res) {
+let errorPage;
+
+export async function fetchErrorPage(fallbackUrl) {
+  const response = await fetch(fallbackUrl);
+  const contentType = response.headers.get('content-type');
+  const contentLength = response.headers.get('content-length');
+
+  // If the Content-Type is not text/html throw an error
+  if (!contentType.includes('text/html')) {
+    throw new Error('Content-Type was not of type text/html');
+  }
+  // If the content length is over 50kb throw an error
+  if (contentLength > 50000) {
+    throw new Error('Content-Length was over 50Kb');
```

Comment:
maybe add a little more info about this for user in the error and does this really matter ?

Reply:
@mtomcal mentioned that we should add a limit on the size of the page. We just didn't know what size to go with so we picked 50kb

Reply:
Shouldn't we prevent abuse of this API? Set a sufficiently large enough HTML file and it could go OOM for the server and sink it.

Reply:
Definitely add some context to these errors `throw new Error('[appConfig/setErrorPage] Content-Length was over 50Kb');`

Reply:
@mtomcal and @JAdshead I have added Tomcal's suggestion. Let me know if any more information should be added.